assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js

```diff
@@ -60,6 +61,9 @@ export default function Footer( {
 	onNavigationToOAuthURL = () => {},
 } ) {
 	const viewContext = useViewContext();
+	const isConversionReportingEnabled = isFeatureEnabled(
+		'conversionReporting'
+	);
```

Comment:
Why isn't this using the `useFeature` feature flag? 🤔

Reply:
Oh indeed, I mistakenly used `isFeatureEnabled`. Fixed